index.html

```diff
@@ -102,6 +118,104 @@ <h3 class="jobHead">software developer</h3>
           <div class="workTxt">Work</div>
         </a>
       </section>
+      <section id="projectCards">
+        <h1 class="nameHead">My recent work</h1>
+        <hr />
+        <div class="cardContainer">
+          <div class="card">
+            <div class="imgplaceholder">g</div>
+            <h3 class="cardTxt">Multi-Post stories</h3>
+
+            <p class="cardText">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="longGradient"></div>
+            <div>
+              <ul class="skillsbar">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>ruby</li>
+              </ul>
+            </div>
+            <div class="probtn">
+              <button class="projectbtn">see project</button>
+            </div>
```

Comment:
- [ ] It will be nice to add button styling: Active, Hover, and Pressed. ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10))

Reply:
@wandji20  I added the pseudo classes for Active and Hover. But Active and pressed seem to mean the same thing(.element:active)

Reply:
Can you check your live project and verify that these effects are functional. That is not the case for me.

Reply:
I pushed my changes and created a new pull request.